attributevalue/src/main/scala/com/hiya/alternator/syntax/package.scala

```diff
@@ -19,5 +19,9 @@ package object syntax {
     def between[T : ScalarDynamoFormat](lower: T, upper: T): RKCondition.BETWEEN[T] = RKCondition.BETWEEN(lower, upper)
   }
 
-
+  object cond {
+    def attr[T](name: String): ConditionExpression.Path[T] = ConditionExpression.Attr(name)
+    def lit[T: DynamoFormat](literal: T): ConditionExpression[T] = ConditionExpression.Literal(literal)
```

Comment:
Do objects accepted here? I mean this can be a scalar or object. ScalarDynamoFormat contains a proof that it is scalar.

Reply:
Apparently not, fixed.